src/FuelClient.cc

```diff
@@ -986,6 +986,14 @@ bool FuelClient::ParseWorldFileUrl(const common::URI &_fileUrl,
     return false;
   }
 
+  std::vector<std::string> tokens = ignition::common::split(file, "/");
+  std::string fileTemp;
+  for (auto s : tokens)
+    fileTemp = ignition::common::joinPaths(fileTemp, s);
+  if(fileTemp[0] == '/')
```

Comment:
How can we end up in this situation after using `split` above?

Reply:
the loop start with an empty string -> `<empty_string> / <2º string>` -> `/<2ºstring>`

I didn't move this code to `ign-common` because maybe it was the expected behaviour, but I can create a PR fixing this there.

Reply:
PR in ign-common https://github.com/ignitionrobotics/ign-common/pull/201